apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/TransformerConstants.java

```diff
@@ -650,4 +653,44 @@ public enum CurrencyIdentifier {
    * C4DIC Highlight Color</a>.
    */
   public static final String C4DIC_HIGHLIGHTCOLOR = "#3B9BFB";
+
+  /**
+   * Code System URL for C4DIC Additional Insurance Card Information Extension URL <a
+   * href="http://hl7.org/fhir/us/insurance-card/StructureDefinition/C4DIC-AdditionalCardInformation-extension">Extension:
+   * C4DIC Additional Insurance Card Information </a>.
+   */
+  public static final String C4DIC_ADD_INFO_EXT_URL =
+      "http://hl7.org/fhir/us/insurance-card/StructureDefinition/C4DIC-AdditionalCardInformation-extension";
+
+  /**
+   * Value for C4DIC Additional Insurance Card Information Extension <a
+   * href="http://hl7.org/fhir/us/insurance-card/StructureDefinition/C4DIC-AdditionalCardInformation-extension">
+   * C4DIC Additional Insurance Card Information </a>.
+   */
+  public static final String C4DIC_ADD_INFO =
+      "You may be asked to show this card when you get health care services. Only give your personal Medicare "
```

Comment:
I know it's spelled this way on the ticket, but shouldn't it be "healthcare" rather than "health care"?